posts/coder-on-railway/index.md

```diff
@@ -0,0 +1,61 @@
+# Deploy Coder on Railway.app
+
+Coder can be deployed on Railway.app in just a few clicks. This guide will walk you through the process of deploying Coder on Railway.app.
+
+## Quickstart
+
+Start your deployment by clicking the button below. This will take you to Railway.app where you can sign up for a free account and deploy Coder.
+
+[![Deploy on Railway](https://railway.app/button.svg)](https://railway.app/template/cUQ8_P?referralCode=tfH8Uw)
+
+### 1. Fill the environment variables
+
+The current version of the template allows you to set an optional environment variable `CODER_WILDCARD_ACCESS_URL` which will allow you to forward ports from your workspace to a custom wildcard subdomain of the URL you provide. This is useful if you want to access a service running in your workspace from a browser. For example, if you set `CODER_WILDCARD_ACCESS_URL` to `*.coder.example.com`, you can access a service running on port 8080 in your workspace by visiting `8080--<username>--<workspace-name>.coder.example.com`.
+
+### 2. Attach a custom domain
+
+Your Coder deployment will always be accessible at `https://<app-name>.up.railway.app`. If you want to use a custom domain, you can do so by going to Dashboard > Coder (Service) > Settings > Domains and adding your domain and optionally a wildcard subdomain if you specified `CODER_WILDCARD_ACCESS_URL` in the previous step.
```

Comment:
If you think it'd be helpful, you can link to our docs here: https://coder.com/docs/v2/latest/admin/configure#wildcard-access-url